pkg/apis/bootstraptoken/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
let's have this API under pkg/apis/kubeadm/bootstraptoken

Reply:
Done, PTAL.